pkg/sensors/tracing/generickprobe.go

```diff
@@ -577,23 +585,6 @@ func createGenericKprobeSensor(
 		Name:  name,
 		Progs: progs,
 		Maps:  maps,
-		PostUnloadHook: func() error {
-			var errs error
-			for _, id := range ids {
-				gk, err := genericKprobeTableGet(id)
-				if err != nil {
-					errs = errors.Join(errs, err)
-				}
-				if gk.stackTraceMapRef != nil {
-					err = gk.stackTraceMapRef.Close()
-					if err != nil {
-						errs = errors.Join(errs, fmt.Errorf("failed to close map: %v", gk.stackTraceMapRef))
-					}
-					gk.stackTraceMapRef = nil
-				}
-			}
-			return errs
-		},
```

Comment:
so we don't need to worry about the ref to this map being closed it's already done by other code?

Reply:
yes, stackTraceMap goes away when the sensor is unloaded, so it's safe